packages/tools/src/tools/ZoomTool.ts

```diff
@@ -151,9 +151,13 @@ class ZoomTool extends BaseTool {
 
     // If it is a regular GPU accelerated viewport, then parallel scale
     // has a physical meaning and we can use that to determine the threshold
-    const imageData = viewport.getImageData();
+    // Added spacing preset in case there is no imageData on viewport
+    let spacing = [0.5, 0.5, 0.5];
+    if (viewport.getImageData()) {
```

Comment:
You can say `const imageData = viewport.getImageData()` and check on imageData instead of twice calling the `viewport.geteImageData()`